src/commands/commerce/payments/quickstart/setup.ts

```diff
@@ -43,19 +43,17 @@ export class PaymentsQuickstartSetup extends SfdxCommand {
         const paymentGatewayAdapterName = `${selection}Adapter`;
         const paymentGatewayProviderName = `${selection}PGP`;
         const paymentGatewayName = `${selection}PG`;
+        const hubOrgAdminUsername = 'ceo@mydevhub.com';
```

Comment:
These are still hardcoded.  They should be sourced from the devhub config.  @tarcang Can you or someone from the team advis @hdogan-sf on how to do this?

Reply:
actually store name and scratch org username can potentially come as command input.